src/SSHDebugPS/ConnectionManager.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.SSHDebugPS.VS;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using System;
+using System.Globalization;
+using System.Linq;
+using liblinux;
+using liblinux.Persistence;
+
+namespace Microsoft.SSHDebugPS
+{
+    internal class ConnectionManager
+    {
+        internal static Connection GetInstance(string name, ConnectionReason reason)
+        {
```

Comment:
>{ [](start = 8, length = 1)

We should still have a TODO comment about keeping a collection of connections and reusing them.